index.html

```diff
@@ -460,12 +460,18 @@
 not be taken as an indication that privacy is always more important than other ethical web principles, and
 this document doesn't address how to balance the different ethical web principles if they come into conflict.
 
-Privacy is covered by legal frameworks and this document recognises that existing data
-protection laws take precedence on legal matters. However, because the Web is global, we
-benefit from having shared concepts to guide its evolution as a system built for the people who use it
-([[?RFC8890]]). A clear and well-defined view of privacy on the Web, informed by research,
-can hopefully help all the Web's participants in different legal regimes. Our shared
-understanding is that the law is a floor, not a ceiling.
+Privacy on the Web is primarily regulated by two forces: the architectural capabilities that the Web
```

Comment:
[[regulated]] cite to Code and Other Laws of Cyberspace, Lawrence Lessig (1999) https://lessig.org/product/code/

Reply:
Done, thanks!

Reply:
FWIW, he first outlined this AFAIK in 'The New Chicago School', and it's a much shorter read :) See https://www.journals.uchicago.edu/doi/pdf/10.1086/468039